tests/e2e-leg-6/verify-vrpq-conditions/40-create-vrpq-with-start-timestamp-filter.yaml

```diff
@@ -15,5 +15,5 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - script: |-
-      export STARTTIMESTAMP=$(kubectl get vrpq archive-query -o json -n $NAMESPACE | jq -r '.status.restorePoints[] | select(.index==1 and .archive=="db") | .timestamp')
+      export STARTTIMESTAMP="$(kubectl get vrpq archive-query -o json -n $NAMESPACE | jq -r '.status.restorePoints[] | select(.index==1 and .archive=="db") | .timestamp')"
```

Comment:
@spilchen  There is one thing I've noticed: when I run it without double quotes on my local machine, it doesn't work. It freezes, possibly due to the timestamp containing a space. I'm unsure why it passes in the GitHub CI but not on my local machine. After discussing with @jizhuoyu , we concluded that adding double quotes would make it more persistent and reliable